ui/src/app/auth/register.animation.ts

```diff
@@ -0,0 +1,15 @@
+import { animate, keyframes, state, style, transition, trigger } from "@angular/animations";
+
+export function registerScreenAnimation(){
+    return  trigger('registerScreenAnimation', [
```

Comment:
debug here and see if this function called multi times

Reply:
called once when I press the register button when I have something wrong with the form